tools/ktx/command_create.cpp

```diff
@@ -783,6 +794,13 @@ Create a KTX2 file from various input files.
                 wrap | reflect | clamp.
                 Defaults to clamp.</dd>
         </dl>
+        <dt>\--normalize</dt>
+        <dd>Normalize input normals to have a unit length. Only valid for
+                linear normal textures with 2 or more components. For 2-component inputs
+                2D unit normals are calculated. Do not use these 2D unit normals
+                to generate X+Y normals with @b --normal-mode. For 4-component inputs
+                a 3D unit normal is calculated. 1.0 is used for the value of the
+                4th component.</dd>
```

Comment:
Add `Cannot be used with @b \--raw.` and make sure you test for that in OptionsCreate::process.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Software/pull/977/commits/29ed55299d6a4328daa1ad3a01e72c5fe963d9b6